tests/unit/worker/worker_test.py

```diff
@@ -296,3 +299,63 @@ def test_get_jobs(zeebe_worker, task):
                                                                 max_jobs_to_activate=task.max_jobs_to_activate,
                                                                 variables_to_fetch=task.variables_to_fetch,
                                                                 request_timeout=zeebe_worker.request_timeout)
+
+
+
+@pytest.fixture
+def stop_after_test():
+    stop_test = Event()
+    yield stop_test
+    stop_test.set()
+
+
+@pytest.fixture
+def handle_task_mock():
+    with patch("pyzeebe.worker.worker.ZeebeWorker._handle_task") as mock:
+        yield mock
+
+
+@pytest.fixture
+def stop_event_mock(zeebe_worker):
+    with patch.object(zeebe_worker, "stop_event") as mock:
+        yield mock
+
+
+@pytest.fixture
+def handle_not_alive_thread_spy(mocker):
+    spy = mocker.spy(ZeebeWorker, "_handle_not_alive_thread")
+    yield spy
```

Comment:
Move these to `conftest.py` so they are with the other fixtures.

Reply:
Addressed in 2c5d96e29413a21e740702bf58d693228b635a40